certbot-apache/tests/autohsts_test.py

```diff
@@ -42,8 +42,11 @@ def get_autohsts_value(self, vh_path):
     @mock.patch("certbot_apache._internal.configurator.ApacheConfigurator.restart")
     @mock.patch("certbot_apache._internal.configurator.ApacheConfigurator.enable_mod")
     def test_autohsts_enable_headers_mod(self, mock_enable, _restart):
-        self.config.parser.modules.discard("headers_module")
-        self.config.parser.modules.discard("mod_header.c")
+        try:
+            del self.config.parser.modules["headers_module"]
+            del self.config.parser.modules["mod_header.c"]
+        except KeyError:
+            pass
```

Comment:
it shouldn't matter because we explicitly add them in the test setup, but I agree that the other way is more robust to changes so I'll switch to that.